pkg/csi/service/node.go

```diff
@@ -503,23 +503,14 @@ func (s *service) NodeGetVolumeStats(
 	*csi.NodeGetVolumeStatsResponse, error) {
 	ctx = logger.NewContextWithLogger(ctx)
 	log := logger.GetLogger(ctx)
-	log.Infof("NodeGetVolumeStats: called with args %+v", *req)
+	log.Debugf("NodeGetVolumeStats: called with args %+v", *req)
```

Comment:
We always maintain the first log to an API as Info. Why is this changed?

Reply:
Changed back to Info.